app/assets/stylesheets/publisher.scss

```diff
@@ -98,6 +98,12 @@
         line-height: $line-height-base !important;
       }
 
+      .twitter-typeahead {
+        position: absolute !important;
```

Comment:
In this case you can disable the linter temporarily and add a comment which gives a reason why we need the `!important` here. See https://github.com/diaspora/diaspora/blob/develop/app/assets/stylesheets/forms.scss#L1-L21 and https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#importantrule